src/call-context.ts

```diff
@@ -0,0 +1,457 @@
+import { PluginOutput, type PluginConfig } from './interfaces.ts';
+import { CAPABILITIES } from 'js-sdk:capabilities';
+
+export const BEGIN = Symbol('begin');
+export const END = Symbol('end');
+export const ENV = Symbol('env');
```

Comment:
Should this be the updated namespace?

Reply:
I'm working on that in a new PR, it requires updating the example plugins and a few other things.

Reply:
In this case `env` doesn't need to be changed – this is just a JS `Symbol` to provide protected, package-only access to the extism host functions.